owners/index.js

```diff
@@ -62,30 +62,37 @@ module.exports = app => {
   listen(
     ['pull_request.opened', 'pull_request.ready_for_review'],
     async (github, payload) => {
-      const pr = PullRequest.fromGitHubResponse(payload.pull_request);
-      if (!payload.pull_request.draft) {
-        await ownersBot.runOwnersCheck(
-          github,
-          pr,
-          shouldAssignReviewers(payload.pull_request)
-        );
-      }
+      const prPayload = payload.pull_request;
```

Comment:
I just moved the `fromGitHubResponse` call into the function call below, so that the handlers could have a standard `const prPayload = payload...` starting line. Just a readability tweak.